examples/kcp/test/e2e/audit-policy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: audit.k8s.io/v1
+kind: Policy
```

Comment:
is this used or just from debugging?

Reply:
It is used in bootstrapped test servers. I think its boilerplate code from old day. Remove or leave?